packages/period-selector-dialog/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@dhis2/d2-ui-period-selector-dialog",
+  "version": "1.0.0",
+  "description": "Period selector dialog component for DHIS2",
+  "main": "./build/cjs/index.js",
+  "module": "./build/es/index.js",
+  "files": [
+    "build"
+  ],
+  "license": "BSD-3-Clause",
+  "author": "Ilya Nee <ilya@dhis2.org>",
+  "contributors": [
+    "Viktor Varland <viktor@dhis2.org>"
+  ],
+  "peerDependencies": {
+    "react": "^15.0.0 || ^16.0.0",
+    "react-dom": "^15.0.0 || ^16.0.0"
+  },
+  "scripts": {
+    "prebuild": "rimraf ./build/*",
+    "build": "npm run build:commonjs && npm run build:es",
+    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir build/cjs --ignore spec.js",
+    "build:es": "cross-env BABEL_ENV=es babel src --out-dir build/es --ignore spec.js",
+    "build:umd": "cross-env && webpack -p",
+    "watch": "npm run build:es --  --watch",
+    "test-ci": "jest --config=../../jest.config.js packages/favorites-dialog"
+  },
+  "dependencies": {
+    "@material-ui/icons": "^1.1.0",
+    "material-ui-next": "^1.0.0-beta.47",
```

Comment:
We can simply import material-ui v1 as material-ui, as we don't need 2 versions in this package.
All import statements in src should be updated accordingly.